hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMapConfiguration.tsx

```diff
@@ -122,7 +126,7 @@ export default class HeatMapConfiguration extends Component {
     if (params.datum.label !== "") {
         tooltipContent += `<br/>
           File Name:
-          ${params.datum.label ? params.datum.label.split("/").slice(-1) : "no"}
+          ${params.datum.label ? params.datum.label.split("/").slice(-1) : ""}
```

Comment:
What file name is this ? and why removed "no", pls explain.

Reply:
@devmadhuu 
If lastnode if label was  not present then dont display label in tooltip. "no" just testing string for negative test cases. so corrected in this PR.